### PR TITLE
Fix spelling mistakes throughout 1.0 documentation

### PR DESCRIPTION
**Pull request by wgraysen**
I've been reading through the Getting Started, Build Images, Languages, Mobile Platforms and How-To sections of the 1.0 documentation as part of my training. I noticed a few spelling mistakes while reading through the documents and thought I'd contribute to fix them. See the commit history/diff for which specific words were corrected and in what files. 😸

### CODE REVIEW

File: jekyll/_cci1/ios-builds-on-os-x.md [hunk 1]
```diff
@@ -317,7 +317,7 @@ your organization's settings page.
 
 ![](  {{ site.baseurl }}/assets/img/docs/fabric-org-settings-page.png)
 
-Click on your organisation (CircleCI in the image above), and click on
+Click on your organization (CircleCI in the image above), and click on
```
Comment (felicianotech) [inline jekyll/_cci1/ios-builds-on-os-x.md]: This isn't a spelling error. Both are acceptable.

Comment (felicianotech) [inline jekyll/_cci1/ios-builds-on-os-x.md]: Personally, I prefer the spelling with the 'z' though. :p

Comment (wgraysen) [inline jekyll/_cci1/ios-builds-on-os-x.md]: As mentioned in the extended comment of my commit, I said:

>Fix localization mistake. Documentation uses US English spelling and in this document we were using 'organisation' not 'organization'. Organisation is primarily only used in UK English.

To quote http://writingexplained.org/organisation-vs-organization-difference, Organisation is the correct spelling in British English and organization is the correct spelling in American English. However, the opposite isn't true: Organisation isn't the correct spelling in American English and Organization isn't the correct spelling in British English.

Comment (felicianotech) [inline jekyll/_cci1/ios-builds-on-os-x.md]: > As mentioned in the extended comment of my commit, I said:

I think you mean typed.

**Comment by wgraysen** [comment]
Thanks for approving the PR! I just added a few of the people that GitHub suggested that I get reviews from since they had edited the file recently.

I'm going to assume that two OKs are enough to merge?

**Pull request merged**